tauri/src/components/container.vue

```
@@ -1,30 +1,82 @@
     console.error('Container ref is not defined')
     return
   }
 
-  const horizontalRuler = new Ruler(container, {
-    type: 'horizontal',
+  const horizontalGuides = new Guides(horizontal.value!, {
+    snapThreshold: 5,
+    snaps: [0, 300, 600],
+    displayDragPos: true,
+    dragPosFormat: v => `${v}px`,
+  }).on('changeGuides', ({ guides }) => {
```

ESLint

[error] 'guides' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)